community/csv/CHANGES.txt

```diff
@@ -1,3 +1,7 @@
+2.2.2
+-----
+o BufferedCharSeeker has ability to allow/disallow multi-line fields
```

Comment:
If I'm reading the code right, this PR changes the behaviour from accepting newlines to throwing and thus changing behaviour. In either case, I think it would be helpful to state what the default is.